src/Language/PureScript/CST/Layout.hs

```diff
@@ -166,6 +166,9 @@
 --    "body of a case of expression" by pushing 'LytOf' onto the layout stack.
 --    Insert the @of@ token into the stream of tokens.
 --
+
+{-# LANGUAGE DeriveAnyClass #-}
+
```

Comment:
Nitpick, I know, but can we get rid of this extra whitespace? The line above/below the language pragma?